orte/mca/errmgr/dvm/errmgr_dvm.c

```diff
@@ -639,7 +774,7 @@ static void proc_errors(int fd, short args, void *cbdata)
          * hosed - so just exit out
          */
         if (ORTE_PROC_MY_NAME->jobid == proc->jobid) {
-            ORTE_ACTIVATE_JOB_STATE(NULL, ORTE_JOB_STATE_DAEMONS_TERMINATED);
+            //ORTE_ACTIVATE_JOB_STATE(NULL, ORTE_JOB_STATE_DAEMONS_TERMINATED);
```

Comment:
Add a nice comment here about what is happening and why we are not doing the default anymore.